.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -1,12 +1,28 @@
 name: 🐞 Bug Report
 description: Please provide a detailed description and include all relevant information about the issue. If you have any doubts, consult our documentation.
+labels:
+  - KendoReact
 body:
 - type: textarea
   attributes:
     label: Description
     description: Add a concise description of the issue and any relevant information.
   validations:
     required: true
+- type: dropdown
+  attributes:
+    label: Product
```

Comment:
Not actually 'Product', I think we call it Plan as stated in our pricing article https://www.telerik.com/kendo-react-ui/pricing 

Product Plan?